tests/selenium/src/lib.mjs

```diff
@@ -263,3 +269,49 @@ export async function sendKeysBy(by, keys) {
         { tag: `sendKeysBy (${by.value || by}) (${keys})` }
     );
 }
+
+export function getTracks() {
+    const tracks = [];
+    // convert mask (support only * as wildcard, eg *wiki*.gpx)
+    const regexp = '*.gpx'.replaceAll('.', '\\.').replaceAll('*', '.*');
+
+    readdirSync('gpx')
+        .sort()
+        .forEach((file) => {
+            if (file.match(/\.gpx$/i) && (!'*.gpx' || file.match(regexp))) {
+                const name = file.replace(/\.gpx$/i, '');
+                const path = resolve('gpx', file);
+                tracks.push({ file, name, path });
+            }
+        });
+    return tracks;
+}
+
+export async function uploadCloudTracks({ files }) {
+    const uploader = async () => {
+        const element = await waitBy(By.id('se-upload-cloud-gpx'));
+        await element.sendKeys(files);
+        return true;
+    };
+
+    await enclose(uploader, { tag: 'Upload cloud track' });
+}
+
+export async function createFolder(name) {
+    await clickBy(By.id('se-add-folder'));
+    const input = await waitBy(By.id('se-add-folder-input'));
+    await input.sendKeys(`${name}`);
+    await clickBy(By.id('se-add-folder-submit'));
+    await waitBy(By.id(`se-menu-cloud-${name}`));
+}
+
+export async function littleTimeout() {
+    await driver.manage().setTimeouts({ implicit: TIMEOUT_OPTIONAL });
+}
+
+export async function deleteTrack(name) {
+    await clickBy(By.id(`se-actions-${name}`));
+    await clickBy(By.id('se-delete-cloud-track'));
+    await clickBy(By.id('se-delete-track-dialog'));
+    littleTimeout().then();
+}
```

Comment:
lib.mjs is a wrapper to Selenium webdriver. And no more.

Lib should work with driver calls only and should be project-independent.

Please move all track/folder functions to distinct action(s) or even to test(s).

Additionally, please avoid using littleTimeout() or any other explicit timeouts.

Any delays should be always based on activity/idle, so you would better use actionIdleWait() to make delay in case when it is really required.